test.js

```diff
@@ -52,7 +52,7 @@ test('store a log line', (t) => {
 test('store an deeply nested log line', (t) => {
   t.plan(4)
 
-  const instance = elastic({ index, type, consistency, host, port })
+  const instance = elastic({index, type, consistency, host, port})
```

Comment:
That is not really a good answer. I think you passed the code through a formatter, right? Changing unneeded lines of code just confuses the history of git. Both syntaxes are valid for standard.